selectors-4/Overview.bs

```diff
@@ -988,6 +992,13 @@ Invalid Selectors and Error Handling</h3>
 <h2 id="logical-combination">
 Logical Combinations</h2>
 
+	Selector logical can be manipulated by
```

Comment:
Should this be "Selector logic"? I don't understand an adjective here.

Reply:
yes, that was a typo :) I'll fix it